pkgs/development/python-modules/databricks-connect/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, isPy37, pkgs, buildPythonPackage, fetchPypi, six, py4j }:
+
+buildPythonPackage rec {
+  pname = "databricks-connect";
+  version = "7.1.0";
+  sha256 = "996a9d0f271f6c7edbd2d85b2efb6ff4e58d15222e80f87ca17fdbf224e17056";
+
+  disabled = !isPy37;
+
+  src = fetchPypi { inherit pname version sha256; };
+
+  sourceRoot = ".";
```

Comment:
What's the motivation for this exactly?

Reply:
I was thinking that is look cleaner. But have seen now that is is normal to have sha256 hash in fetchPypi. So have now changed to the standard way.

Reply:
I mean the sourceRoot = "."

Reply:
Unpacking makes multiple directories. So have to set sourceRoot = "." to tell nix build that we are inside src folder.